force-app/test/package/classes/unit/ArgumentTest.cls

```diff
@@ -75,18 +75,19 @@ public class ArgumentTest {
   }
 
   @isTest
-  static void givenEqualsMatchable_matchesCustomType() {
+  static void givenEqualsMatchable_matchesCustomTypeWithEquals() {
     // Arrange
     Argument.Matchable sut = Argument.equals(new CustomTypeWithEquals('test', 10, new Account(Name = 'test')));
 
     // Act & Assert
     Assert.isTrue(sut.matches(new CustomTypeWithEquals('test', 10, new Account(Name = 'test'))));
-
+    
     Assert.isFalse(sut.matches(null));
     Assert.isFalse(sut.matches(Date.today()));
     Assert.isFalse(sut.matches(DateTime.now()));
     Assert.isFalse(sut.matches(Time.newInstance(12, 12, 12, 12)));
     Assert.isFalse(sut.matches(Blob.valueOf('test')));
+    /*
```

Comment:
**remark:** why those assert are commented out ?

Maybe we want them anyway ?
I think like 101 should be changed to `Assert.isTrue` no ?

Reply:
Oops, this was left over from troubleshooting the GACK. I will fix.